canonical-web-and-design/vanilla-framework
# Bring back labels component

## Done

Labels are back to replace non-interactive chips

Fixes #4183

## QA

- Open [demo](https://vanilla-framework-4184.demos.haus/docs/patterns/labels)
- Check labels examples and docs
  - https://vanilla-framework-4184.demos.haus/docs/patterns/labels
- Make sure we use labels in all places where they have been before:
  - check side navigation in vanilla docs
  - check what's new page
  - check side navigation and application layout examples
